graph/collection-properties.md

```diff
@@ -0,0 +1,402 @@
+OData services treat collections of complex types differently than they treat collections of entity types. 
+This is due to the nature of entity types being "individually addressable" (they have some key which uniquely identifies them within the collection) while complex types are not individually addressable. 
+The result of this difference is that it is almost always preferable to use entity types for collections rather than complex types; please see the exceptions [here](#exceptions).
```

Comment:
I think the style guide is normally about brevity rather than politeness...

```suggestion
The result of this difference is that it is almost always preferable to use entity types for collections rather than complex types. See [exceptions](#exceptions) for acceptable usage of complex types in collections.
```